rclpy/src/rclpy/graph.cpp

```diff
@@ -0,0 +1,252 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Include pybind11 before rclpy_common/handle.h includes Python.h
+#include <pybind11/pybind11.h>
+
+#include <rcl/error_handling.h>
+#include <rcl/graph.h>
+#include <rcutils/error_handling.h>
+#include <rcpputils/scope_exit.hpp>
+
+#include <memory>
+#include <string>
+
+#include "rclpy_common/common.hpp"
+#include "rclpy_common/exceptions.hpp"
+#include "rclpy_common/handle.h"
+
+#include "graph.hpp"
+
+
+namespace rclpy
+{
+
```

Comment:
Huh, I could have sworn `cpplint` complained when there was a blank line after a namespace opening, but the test passes locally for me :shrug:

No need to change it. If the linter is happy I'm happy :upside_down_face:

Reply:
Aha! I had not noticed. `ament_uncrustify` seems to be OK with it too.